massgov/mayflower
# Dp 10107 refactor button sizing

- Make current button "small" the default button size and add "large" and "small" variations.
 - Update the button size for location listing CTA to align with default size.

## Related Issue / Ticket

- Dp 10107
- #176 

## Steps to Test
Patternlab developer, please verify this change doesn't impact drupal location listing submit functionality --> https://github.com/massgov/mayflower/pull/177#discussion_r211316381
